service/grails-app/services/org/olf/rs/ReshareApplicationEventHandlerService.groovy

```diff
@@ -178,6 +179,7 @@ public class ReshareApplicationEventHandlerService {
               def validated_state = lookupStatus('PatronRequest', 'REQ_VALIDATED')
               auditEntry(req, req.state, validated_state, 'Request Validated', null);
               req.state = validated_state;
+              patronNoticeService.triggerNotices(req, "new_request");
```

Comment:
This is really buried in here, but it has the request object already parsed out. Should suffice for now? I could see eventually having a tidier generic setup for the common case of "trigger X upon entering state Y"...